CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ For semantic convention package changes, see the [semconv CHANGELOG](packages/se
 
 ### :house: (Internal)
 
+refactor(sdk-trace-base): add `spanProcessors` property in `TracerConfig` interface. [#5138](https://github.com/open-telemetry/opentelemetry-js/pull/5138) @david-luna
+
```

Comment:
let's put that into (Enhancement) and change it to `feat` - since we're adding stuff 🙂